reminder/module.py

```diff
@@ -0,0 +1,465 @@
+            title=_(utx, "Reminder"),
+        )
+
+        if query.author_id != query.remind_id:
+            embed.add_field(
+                name=_(utx, "Reminded by"),
+                value=reminder_user_name,
+                inline=True,
+            )
+        if query.message != "":
+            embed.add_field(
+                name=_(utx, "Message"),
+                value=query.message,
+                inline=False,
+            )
+        embed.add_field(name=_(utx, "URL"), value=query.permalink, inline=True)
+
+        return embed
+
+    async def _send_reminder_list(self, ctx, query):
+        reminders = []
+
+        for item in query:
+            author = await self._get_member(item.guild_id, item.author_id)
+            remind = await self._get_member(item.guild_id, item.remind_id)
+
+            author_name = author.display_name if author is not None else "(unknown)"
+            remind_name = remind.display_name if remind is not None else "(unknown)"
+
+            reminder = ReminderDummy()
+            reminder.idx = item.idx
+            reminder.author_name = author_name
+            reminder.remind_name = remind_name
+            reminder.remind_date = item.remind_date
+            reminder.status = item.status.name
+            reminder.url = item.permalink.replace("https://discord.com/channels/", "")
+
+            reminders.append(reminder)
+
+        table_pages: List[str] = utils.text.create_table(
+            reminders,
+            {
+                "idx": _(ctx, "Reminder ID"),
+                "author_name": _(ctx, "Author"),
+                "remind_name": _(ctx, "Reminded"),
+                "remind_date": _(ctx, "Remind date"),
+                "status": _(ctx, "Status"),
+                "url": "https://discord.com/channels/",
+            },
+        )
+
+        for table_page in table_pages:
+            await ctx.send("```" + table_page + "```")
+
+    async def _remind(self, item: ReminderItem):
+        reminded_user = await self._get_member(item.guild_id, item.remind_id)
+
+        if reminded_user is None:
+            item.status = ReminderStatus.FAILED
+            item.save()
+            await bot_log.warning(
+                item.remind_id,
+                item.guild_id,
+                "Unable to remind user - member out of bot reach.",
+            )
+            return
+
+        tc = i18n.TranslationContext(item.guild_id, item.remind_id)
+
+        embed = await self._get_embed(utx, item)
+
+        try:
+            await reminded_user.send(embed=embed)
+        except (HTTPException, Forbidden):
+            item.status = ReminderStatus.FAILED
+            item.save()
+            await bot_log.warning(
+                item.remind_id,
+                item.guild_id,
+                "Unable to remind user - blocked PM or not enough permissions.",
+            )
+
+        item.status = ReminderStatus.REMINDED
+        item.save()
+
+        await guild_log.info(
```

Comment:
would make this a debug. I want to see the errors of this module. while I don't care about the successes.